homeassistant/scripts/broadlink.py

```diff
@@ -0,0 +1,238 @@
+"""Script to enter Broadlink RM devices in learning mode."""
```

Comment:
This should also be part of your library. You can register a service in your switch platform to make this functionality available through Home Assistant.